modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -3,16 +3,129 @@
+
+Because the functionality was unmaintainable, however, `paste` no longer supports input from Microsoft Word.
+
+The Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is now available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
+
+In conjunction with this, three changes were made to `paste`:
+
+. A `paste` default has changed. The `paste_data_images` option now defaults to `true`.
++
+When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
++
+. The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
++
+The new name is a more accurate reflection of what sort of data the argument passes.
++
+The `content` argument can no longer be used with `mceInsertClipboardContent`. If `content` is used, no data is passed. `mceInsertClipboardContent` only attempts to pass data if the `html` or `text` property is used.
++
+. The, now built-in, `paste` functionality no longer supports pasting from Microsoft Word.
++
+NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[request for maintainers] should someone in the community be interested in taking over maintainance of this particular functionality as a separate plugin. +
+If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
+
+
+=== Table of Contents
+
+As noted in the xref:6.0-release-notes.adoc#new-and-improved-plugins-table-of-contents[_{productname} 6.0 Release Notes_], the `toc` plugin is no longer part of the Free editor.
+
+Table of Contents has been updated and is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+Users of the Free version of {productname} 5.x who rely on the, now deprecated and removed, `toc` plugin, should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to Premium plugins such as https://tiny.cloud/tinymce/features/#productivity[_Table of Contents_].
+
+The new Premium `tableofcontents` plugin also changes the names of several commands.
+
+* `toc` is now `tableofcontents`
+* `tocupdate` is now `tableofcontentsupdate`
+* `toc_class` is now `tableofcontents_class`
+* `toc_depth` is now `tableofcontents_depth`
+* `toc_header` is now `tableofcontents_header`
+
+Any configurations or logic that use the old command names must be updated to the new names after upgrading to {productname} 6.0 and installing the new `tableofcontents` plugin.
+
+NOTE: Although there are no end-user facing changes, as part of the general architectural change, the `tableofcontents` Premium plugin has modernised its output. It uses the ES2020 library and targets ES2018. It also uses https://terser.org[terser] to improve performance.
```

Comment:
@lnewson, just want to double check that this is correct `It uses the ES2020 library and targets ES2018`

Reply:
No it's not, a lot of this needs resyncing with the changes made in our other PRs.